torch/functional.py

```diff
@@ -9,15 +9,15 @@
 
 
 def split(tensor, split_size, dim=0):
-    """Splits the tensor into equally sized chunks (if possible).
+    """Splits the tensor into chunks all of a certain size (if possible).
```

Comment:
This seems a bit awkwardly phrased.

Maybe, "Splits the tensor into sub-tensors of size :attr:`split_size`."